index.bs

```diff
@@ -1533,7 +1533,155 @@ like {{WeakRef}} or {{FinalizationRegistry}},
+For example, the `canParse()` static method of the URL interface. 
+
+```JS
+if (URL.canParse(someURL)) {
+    // Do stuff...
+}
+```
+
+Give stateful interfaces a constructor, if possible.
+Do not add a constructor if a class has no state.
+Doing so is considered *bad practice*, as it is effectively creating a "fake class":
+that is, instances of the interface do nothing that a static method couldn't do.
+[`DOMParser`](https://html.spec.whatwg.org/#domparser) or [`DOMImplementation`](https://dom.spec.whatwg.org/#domimplementation) are examples of fake classes.
+
+<h4 class="no-num no-toc">Provide a serializer to make interface data more accessible</h4>
+
+Add serializers to transform instances of an interface into a form that is expected to be used by many applications.
+
+A `.toJSON()` method allows an instance of an interface to produce a useful JSON serialization.
+A `.toBlob()` method can be used to extract a binary representation of an interface.
+
+This makes object natural to use with APIs.
+For example, [`GeolocationPosition`](https://www.w3.org/TR/geolocation/#dom-geolocationposition)
+provides a `toJSON()` method:
+
+```JS
+const position = await new Promise((resolve, reject) => {
+    navigator.geolocation.getCurrentPosition(resolve, reject);
+});
+
+const message = JSON.stringify({
+    user: userId,
+    time: Date.now(),
+    position, // .stringify() calls .toJSON() automatically
+});
+```
+
+<h4 class="no-num no-toc">Choose a namespace to Avoid “Fake Classes” for Behavior-Only Utilities</h4>
+
+A namespace exists to group a set of static properties and methods.
+A namespace does not have a prototype.
+Examples include the `Math`, `Intl`, `Atomics`, and `Console` objects in JS.
+
+With one or two small static functions, a whole new namespace may be overkill.
+Attaching them to an existing object might be more appropriate.
+
+Conversely, a namespace that grows too large or too broad may need better organization or separate logical partitions.
+
+<h4 class="no-num no-toc">"Pseudo-namespaces"</h4>
+
+As WebIDL interfaces can have attributes,
+it is possible to create "pseudo-namespaces"
+by defining non-constructable interfaces as attributes.
+
+A common example are all the attributes attaches to the `navigator` object.
+The `navigator` object has an interface definition ([`Navigator`](https://html.spec.whatwg.org/#navigator)),
+which itself contains other attributes that gives access to further functionality
+through interface instances.
+
+For example:
+
+* `navigator.credentials` - The `CredentialsContainer` interface of the Credentials Management API.
+* `navigator.geolocation` - The `Geolocation` interface of the Geolocation specification.
+* `navigator.permissions` - The `Permissions` interface of the Permissions specification.
+
+Psuedo-namespaces are useful for when, as a spec author, you need access to "this" particular instance
```

Comment:
I like that I made a spelling mistake here and you didn't notice :)  It's not a very natural word to use.

Still, this probably needs more work.